gnomad_mitochondria/pipeline/add_annotations.py

```diff
@@ -71,177 +77,192 @@ def main(args):  # noqa: D103
     # Define mt path, output directory, subset name
     subset_name = ""
 
-    logger.info("Adding genotype annotation...")
-    mt = add_genotype(mt_path, min_hom_threshold)
+    try:
```

Comment:
only question for now: was there a "try" struct in the code before? what do the logs look like when this fails, and do we know what would cause it to fail ?

Reply:
it wasn't there before, but we sometimes add this as it just makes it easier to grab the log file. if anything fails for various reasons within the main block the log file will be copied to the location you specify.